src/tools/kdb/import.cpp

```diff
@@ -63,16 +63,9 @@ int ImportCommand::execute(Cmdline const& cl)
 
 	KeySet base;
 
-	// TODO: this should not be neccessary, but
-	// applying threeway strategies to a twoway merge is hard
-	base = KeySet();
+	base = existingKeys;
 
-	// TODO: for now we have to position this strategy manually
-	// to avoid meta information loss
-	MetaMergeStrategy metaStrategy(merger);
-	merger.addConflictStrategy(&metaStrategy);
-
-	helper.parseStrategies (cl, merger);
+	helper.configureMerger (cl, merger);
```

Comment:
Felix Berlakovich wrote:
> To be honest I didn't read into all the import stuff again when patching
> this part. However, the tests still pass, so I guess it should be ok. It

I am not aware of any tests that check if "kdb import" merges correctly.

> was the same as merge.cpp even before the merge configuration patch,
> wasn't it?

You told me about the differences of two-way merging which made it hard to 
implement "kdb import" ;)